app/views/search/_search_field.html.erb

```diff
@@ -1,3 +1,29 @@
+<%
+  if @search_term.present?
+    label_text = "<h1>Search results for <span class=\"visuallyhidden\">&ldquo;#{@search_term}&rdquo;</span></h1>"
+  else
+    label_text = "<h1>Search GOV.UK</h1>"
+  end
+%>
+
+<%= render "govuk_publishing_components/components/search", {
+  inline_label: false,
+  label_text: label_text,
+  size: "large",
+  value: @search_term,
+  id: "search-main"
+} %>
+
+<%= hidden_field_tag("show_organisations_filter", params[:show_organisations_filter]) if params[:show_organisations_filter] %>
+<%= hidden_field_tag("filter_manual[]", params[:filter_manual]) if params[:filter_manual] %>
+<%= hidden_field_tag(:debug_score, params[:debug_score]) if params[:debug_score] %>
+<%= hidden_field_tag(:debug, params[:debug]) if params[:debug] %>
+
+<% if @spelling_suggestion %>
+  <p>Did you mean <%= link_to "#{@spelling_suggestion.query}", @spelling_suggestion.link %></p>
+<% end %>
+
+<% if false %>
```

Comment:
This doesn't seem right?

Reply:
Eeep, left in some debug. Sorry, will correct, thanks.